examples/basic_usage/variables.tf

```
@@ -9,6 +9,12 @@ variable "account_ids" {
   type        = list(string)
 }
 
+variable "terraform_state_bucket" {
```

Why not use the same name as is used in the TF module code?
```suggestion
variable "terraform_state_bucket_name" {
```

---

I was trying to be consistent with what this variable is named in all of our other repos ("terraform_state_bucket"), but I suppose it's probably cleaner in this case to be consistent within this one repo.

---

I see.  How bad would it be to make the variable name in this repo consistent with elsewhere?

---

Looks like it would require [around 7 PRs or so](https://github.com/search?q=org%3Acisagov+%22github.com%2Fcisagov%2Fterraform-state-read-role-tf-module%22&type=code), but I don't think it's something we need to do right now.  I went ahead and made the changes for this PR in https://github.com/cisagov/terraform-state-read-role-tf-module/pull/44/commits/496d9362bbe1c31bb43d9dfc86546f019ed402b5.